src/App.js

```diff
@@ -1,19 +1,25 @@
 import React, { Component } from 'react';
-import { BrowserRouter, Route } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 
 import Header from './components/Header';
 import MapCanvas from './components/MapCanvas';
-import SiteList from './components/SiteList';
-import SiteDetail from './components/SiteDetail';
+import DashBoard from './components/DashBoard';
```

Comment:
Here's a nifty trick. In almost every directory, I have an index.js that looks something like the following:


(components/index.js)
```js
export { default as Header } from './Header'
export { default as MapCanvas } from './MapCanvas'
export { default as Dashboard } from './DashBoard'
```

This allows me to replace lines 4-6 above with the following:

```js
import { Header, MapCanvas, DashBoard } from './components'
```


This is a preference of mine, but it's not essential. I believe it's just a bit more organized, but it doesn't add or take anything from the app perfomance.

Reply:
+1

Reply:
not working...